src/internal/internal_gemmA.cc

```diff
@@ -701,10 +787,78 @@ void gemmA< Target::HostTask, std::complex<double> >(
     std::complex<double> beta,  Matrix< std::complex<double> >&& C,
     Layout layout, int priority, int64_t queue_index,
     Options const& opts);
+
+// ----------------------------------------
+// Devices instantiation
```

Comment:
Not Devices, it's HostNest. But usually just put the `// ----------------------------------------` divider and don't label each section. Cf. internal_gemm.cc.

Reply:
Done.